src/components/relayPiifields.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {graphql, useStaticQuery} from 'gatsby';
+
+const query = graphql`
+  query PiiFieldsQuery {
+    allPiiFieldPath {
+      nodes {
+        path
+      }
+    }
+  }
+`;
+
+function PiiField({field}) {
+  const asterisk_flag = field.endsWith('other');
```

Comment:
Another upside of making `src/data/relay_event_pii.txt` a `.json` would be that you could encode this information inside objects in the `.json` instead of having to rely on the field name.

Reply:
it makes sense, although maybe for now, since this works and I already merged the tool it can stay like this, and then in future we can change it. Although I am not sure about this